### PR TITLE
Document Numerics.BigInteger overloaded APIs (Span related)

### PR DESCRIPTION
**Pull request by carlossanlop**
I corrected some whitespaces, not sure if they will get reverted by the compiler.

Area owners:  @tannergooding, @ViktorHofer

### CODE REVIEW

File: xml/System.Numerics/BigInteger.xml [hunk 1]
```diff
@@ -6627,12 +6624,85 @@ value Mod 2 = 0
...
-        <param name="provider">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">A read-only span of characters that contains the number to convert.</param>
+        <param name="style">A bitwise combination of the enumeration values that specify the permitted format of <paramref name="value" />.</param>
+        <param name="provider">An object that provides culture-specific formatting information about <paramref name="value" />.</param>
+        <summary>Converts the representation of a number, contained in the provided read-only span of characters, in a specified style to its <see cref="T:System.Numerics.BigInteger" /> equivalent.</summary>
+        <returns>A value that is equivalent to the number specified in the <paramref name="value" /> parameter.</returns>
+        <remarks>
```
Comment (carlossanlop) [inline xml/System.Numerics/BigInteger.xml]: I copied these remarks from the overload that receives a string in the place of the ReadOnlySpan. I removed the sections that I considered specific for strings, and left others that simply needed "string" to be renamed to either "span" or "`value`".

I did the same with the other long remark for the TryParse equivalent method.

File: xml/System.Numerics/BigInteger.xml [hunk 2]
```diff
@@ -7482,12 +7588,12 @@ The integer value `33022` can be exported in four different arrays:
...
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <param name="format">To be added.</param>
-        <param name="provider">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">The span of characters into which this instance will be written.</param>
+        <param name="charsWritten">When the method returns, contains the length of the span in number of chars.</param>
```
Comment (rpetrusha) [inline xml/System.Numerics/BigInteger.xml]:
```suggestion
        <param name="charsWritten">When the method returns, contains the length of the span in number of characters.</param>
```

File: xml/System.Numerics/BigInteger.xml [hunk 3]
```diff
@@ -7528,11 +7634,12 @@ The integer value `33022` can be exported in four different arrays:
         <Parameter Name="result" Type="System.Numerics.BigInteger" RefType="out" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The representation of a number as a read-only span of characters.</param>
+        <param name="result">When this method returns, contains the <see cref="T:System.Numerics.BigInteger" /> equivalent to the number that is contained in <paramref name="value" />, or zero (0) if the conversion fails. The conversion fails if the <paramref name="value" /> parameter is <see langword="null" /> or is not of the correct format. This parameter is passed uninitialized.</param>
+        <summary>Tries to convert the representation of a number, contained in the provided read-only span of characters, to its <see cref="T:System.Numerics.BigInteger" /> equivalent, and returns a value that indicates whether the conversion succeeded.</summary>
```
Comment (rpetrusha) [inline xml/System.Numerics/BigInteger.xml]:
```suggestion
        <summary>Tries to convert the representation of a number contained in the specified read-only character span to its <see cref="T:System.Numerics.BigInteger" /> equivalent, and returns a value that indicates whether the conversion succeeded.</summary>
```

File: xml/System.Numerics/BigInteger.xml [hunk 4]
```diff
@@ -7643,13 +7750,83 @@ The integer value `33022` can be exported in four different arrays:
...
-        <param name="provider">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The representation of a number as a read-only span of characters.</param>
+        <param name="style">A bitwise combination of enumeration values that indicates the style elements that can be present in <paramref name="value" />. A typical value to specify is <see cref="F:System.Globalization.NumberStyles.Integer" />.</param>
+        <param name="provider">An object that supplies culture-specific formatting information about <paramref name="value" />.</param>
+        <param name="result">When this method returns, contains the <see cref="T:System.Numerics.BigInteger" /> equivalent to the number that is contained in <paramref name="value" />, or <see cref="P:System.Numerics.BigInteger.Zero" /> if the conversion failed. The conversion fails if the <paramref name="value" /> parameter is <see langword="null" /> or is not in a format that is compliant with <paramref name="style" />. This parameter is passed uninitialized.</param>
```
Comment (rpetrusha) [inline xml/System.Numerics/BigInteger.xml]:
```suggestion
        <param name="result">When this method returns, contains the <see cref="T:System.Numerics.BigInteger" /> equivalent to the number that is contained in <paramref name="value" />, or <see cref="P:System.Numerics.BigInteger.Zero" /> if the conversion failed. The conversion fails if the <paramref name="value" /> parameter is an empty character span or is not in a format that is compliant with <paramref name="style" />. This parameter is passed uninitialized.</param>
```

File: xml/System.Numerics/BigInteger.xml [hunk 5]
```diff
@@ -7528,11 +7634,12 @@ The integer value `33022` can be exported in four different arrays:
         <Parameter Name="result" Type="System.Numerics.BigInteger" RefType="out" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The representation of a number as a read-only span of characters.</param>
+        <param name="result">When this method returns, contains the <see cref="T:System.Numerics.BigInteger" /> equivalent to the number that is contained in <paramref name="value" />, or zero (0) if the conversion fails. The conversion fails if the <paramref name="value" /> parameter is <see langword="null" /> or is not of the correct format. This parameter is passed uninitialized.</param>
```
Comment (rpetrusha) [inline xml/System.Numerics/BigInteger.xml]:
```suggestion
        <param name="result">When this method returns, contains the <see cref="T:System.Numerics.BigInteger" /> equivalent to the number that is contained in <paramref name="value" />, or zero (0) if the conversion fails. The conversion fails if the <paramref name="value" /> parameter is an empty characters span or is not of the correct format. This parameter is passed uninitialized.</param>
```

File: xml/System.Numerics/BigInteger.xml [hunk 6]
```diff
@@ -7643,13 +7750,83 @@ The integer value `33022` can be exported in four different arrays:
...
+        <summary>Tries to convert the string representation of a number to its <see cref="T:System.Numerics.BigInteger" /> equivalent, and returns a value that indicates whether the conversion succeeded.</summary>
+        <returns><see langword="true" /> if <paramref name="value" /> was converted successfully; otherwise, <see langword="false" />.</returns>
+        <remarks>
+          <format type="text/markdown">
+  <![CDATA[
+
+## Remarks
+
+The <xref:System.Numerics.BigInteger.TryParse%28System.ReadOnlySpan%3CSystem.Char%3E%2CSystem.Globalization.NumberStyles%2CSystem.IFormatProvider%2CSystem.Numerics.BigInteger%40%29> method is like the <xref:System.Numerics.BigInteger.Parse%28System.ReadOnlySpan%3CSystem.Char%3E%2CSystem.Globalization.NumberStyles%2CSystem.IFormatProvider%29> method, except that it does not throw an exception if the conversion fails. This method eliminates the need to use exception handling to test for a <xref:System.FormatException> if `value` is invalid and cannot be parsed successfully.
```
Comment (rpetrusha) [inline xml/System.Numerics/BigInteger.xml]:
```suggestion
This overload is like the <xref:System.Numerics.BigInteger.Parse(System.ReadOnlySpan%7BSystem.Char%7D,System.Globalization.NumberStyles,System.IFormatProvider)> method, except that it does not throw an exception if the conversion fails. This method eliminates the need to use exception handling to test for a <xref:System.FormatException> if `value` is invalid and cannot be parsed successfully.
```
I think that this should fix the broken xref, hopefully.

**Comment by BillWagner** [comment]
@carlossanlop I applied Ron's suggestions, and I think everything is good. If you're happy with this now, I'll :shipit:

**Comment by carlossanlop** [comment]
@BillWagner :shipit: 👍

**Pull request merged**